GxTopicMetaInformation topicMetaInfo
string outputTopic
---
bool completed